Packs/Phishing/ReleaseNotes/3_5_8.md

```diff
@@ -0,0 +1,15 @@
+
+#### Layouts
+
+##### Phishing Incident v3
+
+- Added "Selected indicators to block" section and incident field. this section will show the indicators chosen to be blocked during the investigation.
+
+#### Playbooks
+
+##### Phishing - Generic v3
+
+- "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" and the conditional task "Should indicators be blocked automatically?" has been removed. Block indicators method will be determined in the sub-playbook itself using the "AutoBlockIndicators" input.
+- "BlockIndicators" input will now determine whether to automatically block indicators or selecting manually which indicators to block, if any.
+- "Manually search & delete emails" manual task has been added and will be used when the user will choose to not search and delete emails automatically.
```

Comment:
- "Manual Remediation" section and "Manually remediate the incident" task has been removed. Instead, the "Manually search & delete emails" manual task has been added and will be used when the user will choose to not search and delete emails automatically.

(combine the last 2 items into 1)